microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -125,57 +133,85 @@ Define the specific markets in which you plan to offer your extension.  The defa
+| **Website URL** | - | A webpage that provides additional information about your extension.  The `Website URL` must point to a webpage on your own website, not the web listing for your extension in the [Microsoft Edge Add-ons website](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home).  The `Website URL` helps users learn more about your extension, its features, and any other relevant information. |
+| **Support contact details** | - | The URL to your support webpage, or the email address to contact your support team. |
+| **Mature content** | - | Checkbox to specify if your extension includes mature content.  Extension rating helps determine the appropriate age group of the target audience of your extension.  To help determine if your extension has mature content, see [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies). |
 
-Select **Save & Continue** to continue to the **Store listings** section.
+Properties that are not indicated as Required are Optional.
 
-> [!IMPORTANT]
-> Your developer/organization name, website URL, and support contact details that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+Your developer/organization name,<!-- todo: what field name? --> **Website URL**, and **Support contact details** that you submitted<!-- todo: before, or in this form? --> during registration are displayed to users on the Microsoft Edge Add-ons website.
+
+After filling in the **Properties** webpage for the extension, click the **Save & Continue** button.
+
+The **Store listings** section opens.<!-- todo: a webpage in this context, or section of webpage (which webpage & section)? -->
 
 
 <!-- ========================================================================== -->
 ## Step 5: Add Store listing details for your extension
 
-The information provided in the following section is displayed to users who review your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.  To list your extension in the store, the following details are required.
+The information provided in the **Store listings** webpage<!-- todo: a webpage in this context, or section of webpage (which webpage & section)? --> is displayed to users who view your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.
```

Comment:
"The **Store listings** webpage" - in this context, is this a webpage, or a section of a webpage?  which webpage & section?